resources/views/calendars/_calendar.blade.php

```diff
@@ -2,12 +2,6 @@
 /** @var \App\Renderers\CalendarRenderer $renderer
  * @var \App\Models\Calendar $model
  */
-if ($model->missingDetails()): ?>
```

Comment:
If I create a calendar in the quick creator with 0 months, showing the calendar now crashes without this check